manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -4410,6 +4410,17 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "watch", "list"]
+
+  # Maistra specific
+  - apiGroups: ["maistra.io"]
```

Comment:
these changes will need an accompanying PR to maistra/istio-operator that removes [the same lines in build/patch-charts.sh ](https://github.com/maistra/istio-operator/blob/d2588c7e6096377518dc1b98bcab52fec689686d/build/patch-charts.sh#L174-L185)

Reply:
Yes, I remember about that. I didn't submit a pull request yet, because of release and code freeze.

Reply:
you can submit the PR and label it as `do-not-merge`

Reply:
Done: https://github.com/maistra/istio-operator/pull/873